packages/manager/modules/pci/src/projects/new/components/pci-new-project-promotion-banner/template.html

```diff
@@ -0,0 +1,12 @@
+<div class="pci-new-project-promotion-banner" data-ng-if="$ctrl.voucher.valid">
+    <oui-tile
+        class="promotion-banner"
+        data-heading="{{ 'pci_project_promotion_banner_title' | translate }}"
+    >
+        <h5
+            data-translate="pci_project_promotion_banner_promotion"
+            data-translate-values="{ credit: $ctrl.voucher.credit.text }"
+        ></h5>
```

Comment:
i am not sure is intended element, they are used to structure a document and they have accessibility meaning. 
If you use `h5` element you have also to use `h1 > h2 ... >, until h5`, because we have to avoid using heading element by skipping other elements.

you probably want to manage text size ?
For that you can use this: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-content-headings--style

Reply:
The banner is not a structured document with chapters, sections and paragraph so it's ok to represent it with h5 without having to go through the previous bigger sizes. 

Semantically speaking, the banner title is still a header so it shouldn't be presented as a normal text (span or paragraph) with a big font weight. For accessibility reasons it needs to be an HTML header.
The link that you've put in the comment is still about the headers (h1, h2, ...) not just the text size.